### ISSUE TITLE
Funnel math appears incorrect

### ISSUE THREAD

**Issue by jumploops** [mckn/mckn-funnel-panel#17]
Hi there! First off, thanks for your work on this! 
  
# tl;dr 
We have 4 values: `228, 227, 38, and 35`. We're expecting the funnel to show `99.6%, 16.7% and 15.4%` across the steps, but instead we're seeing `99.5%, 1.56%, and 0%`.

## Background

We're trying to use this plugin to do some funnel analysis on our product, but realized the data we're expecting to see isn't showing up correctly. We've simplified our funnel to 4 states, and simplified the values as well in an attempt to get to the bottom of this. 
  
We're loading in 4 Label-Value pairs that look like this: 
![Screenshot 2024-01-03 at 10 45 07 AM](https://github.com/mckn/mckn-funnel-panel/assets/8540079/ed4079eb-a732-4834-b191-f940376b3d72)

When graphing these, we're seeing an unexpected drop off: 
![Screenshot 2024-01-03 at 10 47 54 AM](https://github.com/mckn/mckn-funnel-panel/assets/8540079/efd955a4-f6cd-4461-89f3-aafb897e87e5)

For reference, this is what we're expecting to see from the source data (in Posthog):
![Screenshot 2024-01-03 at 10 49 01 AM](https://github.com/mckn/mckn-funnel-panel/assets/8540079/6e618e1c-1fc8-45a5-b8ec-489c478e51ab)

Any idea what we're doing wrong here? It seems the calculation from 227 to 38 is the most egregious, but all the values seem incorrect.

**Comment by geckogecko** [comment]
I have the same issue. The last value is cut off and the percentage is displayed as 0%.

![Bildschirmfoto 2024-01-15 um 17 18 28](https://github.com/mckn/mckn-funnel-panel/assets/14152307/6a417452-a1c0-49ac-a6bb-bf7493da1f87)

**Comment by geckogecko** [comment]
I fixed the issue for me by setting _Min in Standard Options_ to **0**. Now the calculated percentage and display of the values is correct!

![Bildschirmfoto 2024-01-15 um 17 30 19](https://github.com/mckn/mckn-funnel-panel/assets/14152307/a9fa5299-ac70-4f46-b511-4829fe17c376)

**Comment by jumploops** [comment]
@geckogecko thank you! This is both obvious and nonobvious at the same time 😅

**Comment by gprashmi** [comment]
I had the same issue and solved it based on the comment by @geckogecko. Thank you for the solution!

**Comment by Noshire** [comment]
Thanks for the great plugin! I just wanted to chime in here. I also got confused by this (a delta from 3 to 2 gives me "-50%", for example). The suggested fix above worked, but it did take me some time to first realize that there is an issue, narrow down the issue, try to find a workaround in Grafana, and then search for others with the same issue.

I think a lot of headache could be avoided for future users of this plugin by either defaulting Min to 0, or prominently mentioning this behaviour somewhere in the plugin's UI.
